product_docs/docs/edbcloud/beta/getting_started/01_connect_cloud_account.mdx

```diff
@@ -4,15 +4,13 @@ title: "Connecting Your Cloud Account"
 
 This topic describes how to set up your EDB Cloud subscription on Azure Marketplace. Your Azure subscription is where EDB Cloud can create and manage PostgreSQL clusters.
 
-## Before You Begin
+## Before You Create Your Subscription
```

Comment:
I would leave this as "Begin" or something like "Prerequisites for Setting Up EDB Cloud". We aren't going to walk customers through how to create an Azure subscription.